docs/reference/configuration.md

```diff
@@ -28,15 +28,43 @@ This reference uses `.` to denote the nesting of values.
 * `logging.level` `(one of ["TRACE", "DEBUG", "INFO", "WARN", "ERROR", "NONE"] : "DEBUG")` - Logging level to output
-* `logging.audit_log_level` `(one of ["TRACE", "DEBUG", "INFO", "WARN", "ERROR", "NONE"] : "DEBUG")` - Audit logs level to output. **Please notice that in case you configure this field to be lower than the main logger level, you won't be able to get the audit logs**
+* `logging.audit_log_level` `(one of ["TRACE", "DEBUG", "INFO", "WARN", "ERROR", "NONE"] : "DEBUG")` - Audit logs level to output.
+
+  **Note:** In case you configure this field to be lower than the main logger level, you won't be able to get the audit logs 
+  {: .note }
 * `logging.output` `(string : "-")` - A path or paths to write logs to. A `-` means the standard output, `=` means the standard error.
 * `logging.file_max_size_mb` `(int : 100)` - Output file maximum size in megabytes.
 * `logging.files_keep` `(int : 0)` - Number of log files to keep, default is all.
 * `actions.enabled` `(bool : true)` - Setting this to false will block hooks from being executed
-* `database.connection_string` `(string : "postgres://localhost:5432/postgres?sslmode=disable")` - PostgreSQL connection string to use
-* `database.max_open_connections` `(int : 25)` - Maximum number of open connections to the database
-* `database.max_idle_connections` `(int : 25)` - Sets the maximum number of connections in the idle connection pool
-* `database.connection_max_lifetime` `(duration : 5m)` - Sets the maximum amount of time a connection may be reused
+* ~~`database.connection_string` `(string : "postgres://localhost:5432/postgres?sslmode=disable")` - PostgreSQL connection string to use~~
+* ~~`database.max_open_connections` `(int : 25)` - Maximum number of open connections to the database~~
+* ~~`database.max_idle_connections` `(int : 25)` - Sets the maximum number of connections in the idle connection pool~~
+* ~~`database.connection_max_lifetime` `(duration : 5m)` - Sets the maximum amount of time a connection may be reused~~
+
+  **Note:** Deprecated - See `database` section 
+  {: .note }
+* `database` - Configuration section for the lakeFS key-value store database
+  + `database.type` `(string : ["postgres"|"dynamodb"])` - lakeFS database type 
+  + `database.postgres` - Configuration section when using `database.type="postgres"`
+    + `database.postgres.connection_string` `(string : "postgres://localhost:5432/postgres?sslmode=disable")` - PostgreSQL connection string to use
+    + `database.postgres.max_open_connections` `(int : 25)` - Maximum number of open connections to the database
+    + `database.postgres.max_idle_connections` `(int : 25)` - Sets the maximum number of connections in the idle connection pool
+    + `database.postgres.connection_max_lifetime` `(duration : 5m)` - Sets the maximum amount of time a connection may be reused
+  + `database.beta_dynamodb` - Configuration section when using `database.type="dynamodb"`
```

Comment:
we should remove the 'beta' before the merge, right?

Reply:
Will be decided next week, per dynamodb testing results